Frontend/src/Components/BatteryCells/BatteryCells.js

```diff
@@ -1,147 +1,214 @@
-import { Box, Flex, Spacer, SimpleGrid, Heading, Text, VStack } from "@chakra-ui/react";
+import { SimpleGrid, VStack } from "@chakra-ui/react";
+import DataPack from "./DataPack";
 import CellGroup from "./CellGroup";
 
 export default function BatteryCells(props) {
     return (
-
-        <VStack>
+        <SimpleGrid
+            columns={2}
+            rows={1}
+            spacingX={'0.25vw'}
+            h="calc(100% - 26px)" // Calculate height of container (100%) - height of Select (24px) and border (2px)
+        >
+            <SimpleGrid
+                columns={1}
+                overflowY="scroll"

```

Comment:
```suggestion
```
Let's try to fit all of the data within the container without overflow/needing to scroll.